### PR TITLE
ISPN-14822 Add new indexing parameters to wizard

### PR DESCRIPTION
**Pull request by dpanshug**
https://issues.redhat.com/browse/ISPN-14822

### CODE REVIEW

**Comment by andyuk1986** [comment]
@dpanshug  could you please rebase with main as it contains a new commit with some test adjustments.

**Review by dvagnero — COMMENTED** [review]
Hi @dpanshug I added two suggestions. Other than that it looks good to me

File: src/app/assets/languages/en.json [hunk 1]
```diff
@@ -258,6 +258,10 @@
           "eviction-strategy-tooltip": "Specify how {{brandname}} should remove entries when the cache reaches the threshold that you configure. The REMOVE strategy evicts older entries to make space for new ones. The EXCEPTION strategy configures {{brandname}} to log an exception and prevent new entries from being created.",
           "indexed": "Indexing",
           "indexed-tooltip": "To enable faster queries and full-text search, build an index of cache values.",
+          "indexing-mode": "Indexing mode",
+          "indexing-mode-tooltip": "Configure how {{brandname}} builds the index.",
```
[the remark this answers is not in the document]
Comment (andyuk1986) [reply src/app/assets/languages/en.json:262]: Applied.

File: src/app/assets/languages/en.json [hunk 2]
```diff
@@ -269,6 +273,8 @@
           "index-startup-mode": "Startup mode",
           "index-startup-mode-tooltip": "Configure how {{brandname}} builds the index.",
           "index-startup-mode-placeholder": "Select a startup mode",
+          "index-sharding": "Index sharding",
+          "index-sharding-tooltip": "Splitting index data into multiple smaller indexes",
```
[the remark this answers is not in the document]
Comment (andyuk1986) [reply src/app/assets/languages/en.json:277]: Applied.

[the remark this answers is not in the document]
Comment (dpanshug) [reply src/app/assets/languages/en.json]: Thanks @andyuk1986

**Pull request merged**